api/v1alpha1/addresspool_types.go

```diff
@@ -42,10 +42,19 @@ type AddressPoolSpec struct {
 	AutoAssign bool `json:"autoAssign,omitempty"`
 }
 
+// AddressPool Services information
+type AddressPoolSvcStats struct {
+	// Service Name
+	Service string `json:"service,omitempty"`
```

Comment:
Is service name will also contains the namespace of the service in `<namespace>/<service-name>`  or we should add also Namespace here?

Reply:
I will separate them but now single IP can be used by multiple svcs in different namespace so I need both service and namspace to be a list